recipes/raylib/all/conanfile.py

```diff
@@ -72,14 +109,40 @@ def generate(self):
             tc.variables["USE_EXTERNAL_GLFW"] = "ON"
             tc.variables["OPENGL_VERSION"] = "OFF" if not self.options.opengl_version else self.options.opengl_version
         tc.variables["WITH_PIC"] = self.options.get_safe("fPIC", True)
+
+        tc.variables["CUSTOMIZE_BUILD"] = True
+        true_false = lambda x: True if x else False
+        tc.variables["SUPPORT_MODULE_RSHAPES"]   = true_false(self.options.module_rshapes)
```

Comment:
```suggestion
        tc.variables["SUPPORT_MODULE_RSHAPES"]   = self.options.module_rshapes
```

The OptionValue is converted to boolean already, you should not need another helper function.